src/js/utils/media.js

```diff
@@ -57,3 +58,16 @@ export function resizeImage(file) {
         return canvas.toDataURL('image/jpeg');
     });
 }
+
+export function mergeMessages(response, storeMessages, message) {
```

Comment:
I'd put that directly in the reducer file too since it's really specific to that part of the lib. Also the media utils is more about images :)